Resources/views/GraphiQL/index.html.twig

```diff
@@ -24,11 +24,13 @@
   {% block body_loading %}Loading...{% endblock body_loading %}
   {% block body_script %}
   <script>
-    const endpoint = {{ endpoint | json_encode | raw }}
+    var endpoint = {{ endpoint | json_encode | raw }}
 
     function graphQLFetcher(params) {
+      var headers
```

Comment:
Prevent `headers` from being undefined if `graphql_fetcher_headers` is rewritten